src/main/java/ca/uhn/fhir/jpa/starter/dotBase/searchBuilder/Queries.java

```diff
@@ -0,0 +1,70 @@
+package ca.uhn.fhir.jpa.starter.dotBase.searchBuilder;
+
+public class Queries {
+
+  protected static String matchAll(String findFieldName, String searchFieldName) {
+    return (
+      "SELECT r." +
+      findFieldName +
+      " FROM ResourceLink r WHERE r." +
+      searchFieldName +
+      " IN (:target_pids) "
+    );
+  }
+
+  protected static String notMatchAll(
+    String findFieldName,
+    String searchFieldName,
+    String targetResourceType,
+    boolean haveTargetTypesDefinedByParam
+  ) {
+    if (targetResourceType != null) return Queries.withTargetResourceType(
+      findFieldName,
+      searchFieldName
+    ); else if (
+      haveTargetTypesDefinedByParam
+    ) return Queries.haveTargetTypesDefinedByParam(
+      findFieldName,
+      searchFieldName
+    ); else return Queries.NoTargetResourceType(findFieldName, searchFieldName);
```

Comment:
Please use curly brackets to enclose return blocks instead of chaining this if statement.
```suggestion
    if (targetResourceType != null) {
        return Queries.withTargetResourceType(findFieldName, searchFieldName); 
    } else if (haveTargetTypesDefinedByParam) {
        return Queries.haveTargetTypesDefinedByParam(findFieldName, searchFieldName); 
    } else {
        return Queries.NoTargetResourceType(findFieldName, searchFieldName);
    }
```

Reply:
:+1: